Event/include/Event/TrigScintCluster.h

```diff
@@ -0,0 +1,230 @@
+/**
+ * @file TrigScintCluster.h
+ * @brief Class that stores cluster information from the Trigger Scintillator
+ * @author Lene Kristian Bryngemark, Stanford University : adaptation to Trigger Scintillator
+ */
+
+#ifndef EVENT_TRIGSCINTCLUSTER_H_
+#define EVENT_TRIGSCINTCLUSTER_H_
+
+// STL
+#include <iostream>
+#include <set>
+
+// ldmx-sw
+#include "Event/EventConstants.h"
+#include "Event/TrigScintHit.h"
+
+
+namespace ldmx {
+
+  /**
+   * @class TrigScintCluster 
+   * @brief Stores cluster information from the trigger scintillator pads. Adds on the ECal cluster functionality
+   */
+  class TrigScintCluster : {
```

Comment:
I think this is a typo.  I guess we're not intending this to inherit from anything.  The code also won't compile like this.